src/Controllers/MagicLinkController.php

```diff
@@ -163,6 +163,12 @@ public function verify(): RedirectResponse
 
         $this->recordLoginAttempt($identifier, true, $user->id);
 
+        // Give the developer a way to know the user
+        // logged in via a magic link.
+        session()->setTempdata('magic_login', true);
+
+        Events::trigger('magic_login');
```

Comment:
`failedLogin` and `magic_login`, not consistent. `magicLogin`?

Please update the docs:
https://github.com/codeigniter4/shield/blob/develop/docs/events.md#event-list
https://github.com/codeigniter4/shield/blob/develop/docs/session_auth_event_and_logging.md#login